lib/float.gi

```diff
@@ -252,23 +252,21 @@ end);
 
 #############################################################################
 ## Default methods
+##
+## These methods have priority -1, because they are inefficient.
+## Hopefully every float implementation implements them better.
```

Comment:
I don't understand this: If a float implementation implements them better, then it would install its methods for its own filter, which would be a subfilter of `IsRealFloat` or `IsComplexFloat`, and hence would automatically have a higher rank.

Or perhaps you are concerned about float implementations which implement neither IsRealFloat` nor `IsComplexFloat` (such as `float` right now, as `IsRealFloat` or `IsComplexFloat` do not yet exist on `master`) ? But then this should be said so explicitly